en/app.ftl

```diff
@@ -30,7 +30,9 @@
 ## Meta Data
 
 meta-title = { -brand-name-firefox-relay }
+# Deprecated
 meta-description = { -brand-name-firefox-relay } makes it easy to create email aliases, aliases that forward to your real inbox. Use it to protect your online accounts from hackers and unwanted messages.
+meta-description-2 = { -brand-name-firefox-relay } makes it easy to create email masks that forward your messages to your true inbox. Use them to protect your online accounts from hackers and unwanted messages.
```

Comment:
(blocking) suggestion: We want to have a unified (non-numeric) variable name to identify all new strings from this update (instead of `-2`)

Can you adjust all new IDs to have the same appending name? (`-mask`)

```suggestion
meta-description-mask = { -brand-name-firefox-relay } makes it easy to create email masks that forward your messages to your true inbox. Use them to protect your online accounts from hackers and unwanted messages.
```

Reply:
Ah, I didn't realise this was a hard requirement. Could you clarify on why this is so important as to be blocking? We're talking several more hours of _very_ tedious (and therefore error-prone) work, so I want to make sure that that's worth it.

To clarify, I did actually start with calling them `-mask`, but that made some IDs very confusing, since "mask" sounded as if it was part of the description. And of course, we'd get situations like `-2-mask`. Thus, I opted for consistency with our existing method of updating strings.